test/higher_order_ops/test_invoke_subgraph.py

```diff
@@ -768,6 +768,28 @@ def run(x, train=True):
         r1.sum().backward()
         weight.grad.clone()
 
+    def test_return_none_from_fwd(self):
+        @mark_compile_region
+        def gn(x):
+            return x * 2, None, x * 3
+
+        def fn(x):
+            ys = gn(x)
+            return ys[0] + ys[2]
+
+        opt_fn = torch.compile(fn, backend="inductor", fullgraph=True)
```

Comment:
Can we see an expecttest for the graph?

Reply:
Added for both Dynamo and AOT fwd and bwd